src/gfx_func.h

```diff
@@ -139,7 +139,7 @@ bool FillDrawPixelInfo(DrawPixelInfo *n, int left, int top, int width, int heigh
  */
 static inline int CenterBounds(int min, int max, int size)
 {
-	return min + (max - min - size + 1) / 2;
+	return (min + max - size + 1) / 2;
```

Comment:
Technically this is less overflow safe; imagine `min` to be 2^30, max 2^30 + 100 and size 50. The old code would return 2^30 + 25, the new code would return -2^30 + 26. However, the old code is not guaranteed to always return the right value, see e.g. https://www.youtube.com/watch?v=sBtAGxBh-XI for the mess you can get with getting the midpoint of two values.
An even simpler version would be `std::midpoint(min, max - size + 1)` (with C++20).
Having said that, I guess it's outrageous to have such large coordinates to draw with, so I guess in the end it's not a big issue.

Reply:
Yeah, I think at the point we are hitting those possibilities it's well outside the range of being displayable. Though this commit itself is not particularly relevant to the rest.